### PR TITLE
JFS-23 Checkbox formatting

### PR DESCRIPTION
**Pull request by fikander**

### CODE REVIEW

**Comment by mike-lawson** [comment]
@fikander  It looks like we use the tick without a border for the account creation page - can you provide that behavior before we merge this?

**Comment by fikander** [comment]
Do you mean these? https://app.zeplin.io/project/5c75571ebf0d81bce428302d/screen/5c7557417e270e6280ccb284

They are just ticks, not checkboxes. They even use different svg.

File: src/components/Icons/Tick.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    svgElement = (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 16 13"
+        {...svgProps}
+      >
+        <path
+          fill="#737373"
+          fillRule="nonzero"
+          d="M13.5 0l2.121 2.121-10.06 10.061L0 6.622 2.121 4.5l3.44 3.44z"
```
Comment (mike-lawson) [inline src/components/Icons/Tick.js]: From https://github.com/aaa-ncnu/join-frontend-service/pull/28/files#diff-d2c881320e2d95742a05464bf2858aeaR4
``` 
<svg
  {...props}
  xmlns="http://www.w3.org/2000/svg"
  width="16"
  height="13"
  viewBox="0 0 16 13"
>
  <path
    fill="#737373"
    fillRule="nonzero"
    d="M13.5 0l2.121 2.121-10.06 10.061L0 6.622 2.121 4.5l3.44 3.44z"
  />
</svg>
```

Looks like the same SVG, just without the extra border styles.

Comment (fikander) [reply src/components/Icons/Tick.js]: ok

Comment (fikander) [reply src/components/Icons/Tick.js]: actually, it is a different svg with path:
`d="M1.28 4.974L4.635 8.46 12.705 0 14 1.217 4.589 11 0 6.146z"`

compared to this:
`d="M13.5 0l2.121 2.121-10.06 10.061L0 6.622 2.121 4.5l3.44 3.44z"`
 and it's named `invalid-name.svg`

I think it should be a different icon

Comment (mike-lawson) [reply src/components/Icons/Tick.js]: Looks like the desktop and the mobile designs have different assets. We're using tick.svg in mobile, and invalid-name.svg in the desktop version. Let me reach out to Darren to see if that's what he intends.

**Pull request merged**